froide/templates/mfa/auth_recovery.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{% load static i18n %}
+{% load account_tags %}
+{% load form_helper %}
+
+{% block title %}{% translate "Two-factor login" %}{% endblock %}
+
+{% block navbar %}
+  {% include "header_minimal.html" %}
+{% endblock %}
+
+{% block app_body %}
+<h1>
+  {% translate "Two-factor login" %}
+</h1>
+<p>
+  {% translate "Use a recovery code." %}
+</p>
+
+<p>
+  {% blocktranslate %}
+  <strong>WARNING</strong>:
+  The recovery code will be removed after it has been used.
+  Make sure to create a new one after login!
+  {% endblocktranslate %}
+</p>
+
+{% get_mfa_keys as mfa_keys %}
+
+<form method="POST" class="disable-submit">
+  {% csrf_token %}
+  {% for error in form.non_field_errors %}
+      <p>{{ error }}</p>
+  {% endfor %}
+  {% render_field form.code %}
+  <button type="submit" class="btn btn-lg btn-primary">
+    {% translate "Verify code" %}
+  </button>
+  <p class="mt-3">
+    {% if mfa_keys.by_method.FIDO2 %}
+      <a class="btn btn-light" href="{% url 'mfa:auth' 'FIDO2' %}">
+        {% translate "Use hardware security key" %}
+      </a>
+    {% endif %}
+    {% if mfa_keys.by_method.recovery %}
+      <a class="btn btn-light" href="{% url 'mfa:auth' 'recovery' %}">
+        {% translate "Use recovery code" %}
+      </a>
+    {% endif %}
+  </p>
+</form>
+{% endblock %}
```

Comment:
```suggestion
{% endblock %}
```

all other files have a line-ending (is your `editorconfig` /editor properly setup - if you have one?)